src/common/pick_address.cc

```diff
@@ -112,6 +112,13 @@ void pick_addresses(CephContext *cct, int needs)
     exit(1);
   }
 
+  if (!cct->_conf->public_network.empty()
+       && cct->_conf->cluster_network.empty()) {
+    lderr(cct) << "Public network was set, but cluster network was not set " << dendl;
+    lderr(cct) << "    Using public network also for cluster network" << dendl;
+    cct->_conf->set_val("cluster_network", cct->_conf->public_network);
+  }
```

Comment:
I think it would be better if we do this without actually changing the cluster_network option.  Just set up a local value and use that below instead...